docs/docs/spec.md

```diff
@@ -340,9 +340,12 @@ To determine whether a type usage appears in a null-marked scope:
 Iterate over all the declarations that enclose the type usage, starting from the
 innermost.
 
-"Enclosing" is defined as follows: Class members are enclosed by classes, which
-may be enclosed by other class members or classes. Top-level classes are
-enclosed by packages, which may be enclosed by modules.
+"Enclosing" is defined as follows:
+
+-   Class members are enclosed by classes.
+-   Classes may be enclosed by other class members or classes.
```

Comment:
"other class members" refers to local and anonymous classes, right?

Reply:
I think this text predates my involvement, but I was figuring that it meant "methods or constructors," with the cases of local and anonymous classes covered by "or classes."

Does that makes sense? I think this definition could be tightened up in general, but I think tools can largely just lean on the AST representation and get the right behavior as a result. For our immediate purposes, I want to make sure I'm not missing anything big.

Reply:
Sorry. I meant to ask whether the only kinds of classes that are enclosed by class members are local and anonymous ones.

(Of course even that is ambiguous because a nested class is also a class member.)

Reply:
Ah. Yes, I believe so, with the caveat you give about a class nested in an already nested class.